connectors/src/resources/connector_resource.ts

```diff
@@ -0,0 +1,28 @@
+import type { ConnectorProvider } from "@dust-tt/types";
```

Comment:
Why not under storage. Having both resources and storage at the same level seems confusing? Would rather have everything storage related under storage and feels like resources are storage related?